app/controllers/admin_controller.rb

```diff
@@ -1,5 +1,5 @@
 class AdminController < ApplicationController
-  before_filter :require_admin, :except => [:index, :show]
+  before_filter :require_admin, :except => [:index, :show, :edit, :update]
```

Comment:
I can make those changes, but I verified that these changes do not allow any user to get to the 'edit user'/'update user' routes.